sstables/sstables.cc

```diff
@@ -1396,6 +1396,26 @@ void sstable::write_filter() {
     write_simple<component_type::Filter>(filter_ref);
 }
 
+uint64_t sstable::total_components_memory_size() const {
+    return _components->filter ? _components->filter->memory_size() : 0;
```

Comment:
Traditionally node memory is size_t (while on-disk storage is uint64_t).

Reply:
Ack. I have updated the code.